clients/rust-legacy/Cargo.toml

```diff
@@ -33,7 +33,7 @@ spl-record = { version = "0.3.0", features = ["no-entrypoint"] }
 spl-token = { version = "8.0", features = ["no-entrypoint"] }
```

Comment:
Actually, while you're at it, can you also add the `no-entrypoint` feature to the `spl-elgamal-registry` dependency here?

Reply:
Done